src/org/labkey/cds/data/steps/CDSImportTask.java

```diff
@@ -92,7 +92,8 @@
         new TSVCopyConfig("ELISpotAntigen", "AssayELSAntigen_Metadata"),
         new TSVCopyConfig("NAbAntigen", "AssayNABAntigen_Metadata"),
         new TSVCopyConfig("BAMAAntigen", "AssayBAMAAntigen_Metadata"),
-        new TSVCopyConfig("antigenPanel", "AntigenPanel_Metadata"),
+        new TSVCopyConfig("antigenPanelMeta", "AntigenPanel_Metadata"),
```

Comment:
Isn't the file name AssayBAMAAntigen_Metadata.txt ?

Reply:
Correct, it is. I've renamed the table 'antigenPanel' to 'antigenPanelMeta', that change is reflected here.